meta-mender-qemu/recipes-mender/mender-client/mender-client-test-files.inc

```diff
@@ -7,6 +7,14 @@ do_compile_append_mender-image_qemuall() {
         cat > ${WORKDIR}/rootfs-image-v2.conf <<EOF
 MENDER_ROOTFS_PART_A="${MENDER_ROOTFS_PART_A}"
 MENDER_ROOTFS_PART_B="${MENDER_ROOTFS_PART_B}"
+EOF
+    fi
+
+    if [ -d "${WORKDIR}/etc/ssl" ]; then
```

Comment:
I don't think we should hard-code this anywhere, but rather make the client implementation resilient to openssl parameters if possible.

Reply:
@alfrunes I had a discussion with @merlin-northern and we think that making qemu match raspbian is the least we could do (i.e. have SECLEVEL=2 in both places).

If we want to test for multiple configurations, we need to separate this change to a separate layer, and then add it conditionally via fixture for the acceptance tests. Is it worth it? (If so I can take over or mentor @merlin-northern on how to do this).

Reply:
> I don't think we should hard-code this anywhere, but rather make the client implementation resilient to openssl parameters if possible.

client code is one thing (we can do it, check and support other levels, but this is only a discussion about tests! since all will work, if you conform with your settings in your `openssl.cnf`, if you change them, you knew what you were doing and you have to align the keys lengths for instance). the thing here is: to make qemu image reflect what we build and ship with raspbian. and by default we ship `seclevel=2`.